Scripts/Python/plasma/Plasma.py

```diff
@@ -5383,6 +5383,49 @@ def playerSetOnline(self, state):
         """Not sure this should be used. Sets the state of the player online status for this player info node."""
         ...
 
+class ptVaultPlayerNode(ptVaultNode):
+    """Plasma vault player node"""
+
+    def getAvatarShapeName(self):
+        """Returns the avatar's 'shape'"""
+        ...
+
+    def getOnlineTime(self):
+        """Returns the avatar's online time"""
+        ...
+
+    def getPlayerName(self):
+        """Returns the player's name"""
+        ...
+
+    def isDisabled(self):
+        """Is the avatar disabled?"""
+        ...
+
+    def isExplorer(self):
+        """Returns true for 'explorer', false for 'visitor'."""
+        ...
+
```

Comment:
I would go ahead and remove this one too. We instituted a change about a decade ago that removed the concept of visitor accounts.

Reply:
Hmm, this is a thin wrapper around the C++ `VaultPlayerNode`, which still has `Int32_2` mapped to `Explorer`, so I would have kept it... Similarly, `ptVaultAgeInfoNode` has `getCzar`/`getCzarID` even though the "czar" concept doesn't exist anymore.

I'm not hugely opposed to removing it though - not like it will have any practical use in future scripts. (I would keep it on the C++ side though, just so we're aware of the former meaning if we ever want to reuse that field on player nodes.)

Reply:
... I noticed now that the czar methods are stubbed out, so perhaps that's not a meaningful precedent, lol. I've removed `isExplorer`/`setExplorer` now.